docs/guides/firmware-update.md

```diff
@@ -0,0 +1,53 @@
+# Firmware update
+
+## Introduction
+
+This document describes the process for updating firmware on devices running
+Dasharo firmware. Some steps specific to each device will be described in their
+respective documentation, but the generic process outlined here will apply to
+all devices.
+
+## Firmware Update Mode
+
+For enhanced firmware security, Dasharo uses a number of security mechanisms to
+prevent overwriting firmware. Depending on device, these may be some or all of
+the following:
+
+- SPI flash write-protection - prevents overwriting the initial bootblock and
+  Verified Boot code
+- SMM BIOS write protection - prevents all writes to BIOS flash memory outside
+  of privileged code running in System Management Mode
+- UEFI Secure Boot - in combination with Linux Kernel Lockdown, prevents direct
+  access to the SPI flash controller from the OS
+
+If you are interested, see the
+[Dasharo System Features](../dasharo-menu-docs/dasharo-system-features.md)
+section for more details.
+
+To allow updating firmware by the end user, these protections must be disabled
+first. To facilitate this, Dasharo has a Firmware Update Mode option that
+**temporarily** disables firmware security measures for the duration of one
+boot.
+
+To enter Firmware Update Mode:
+
+1. Enter the Setup Menu:
+![](./images/setup_menu_dsf.png)
+1. Navigate to `Dasharo System Features`:
+![](./images/setup_menu_dsc.png)
+1. Navigate to `Dasharo Security Options`:
+![](./images/setup_menu_fum.png)
+1. Select the `Enter Firmware Update Mode` option:
+![](./images/setup_menu_fum_confirmation.png)
+1. When prompted, press Enter to accept. The device will reboot in Firmware
+  Update Mode.
```

Comment:
The images seem to be off-by-one step. In point 2  we say to navigate to Dasharo SYstem Features, while we are already in Dasharo SYstem features on the picture below.

In point 4 we say to select Enter Firmware Update Mode, but in the picture below it we have already selected it and a prompt appeared. The information what to do about prompt is in next point...

I would merge point 1 and 2 and then shift all images one point lower

Reply:
Ok, done.